ocs_ci/helpers/helpers.py

```diff
@@ -2942,6 +2943,28 @@ def get_mon_pdb():
     return disruptions_allowed, min_available_mon
 
 
+def safe_get_attr(obj, attribute):
+    """
+    Safely retrieve object attributes
+
+    Args:
+        obj: An object
+        attribute (str): A string containing one attribute or several sub-attributes
+        separated by dots
+        (i.e. - "cld_mgr.aws_client", "cld_mgr.aws_client.secret")
+
+    Returns:
+        The requested attribute if found, otherwise None
+    """
+    if obj is not None:
+        try:
+            return reduce(getattr, attribute.split("."), obj)
+        except AttributeError:
+            return None
+    else:
+        return None
```

Comment:
Also add few unit tests, with test cases for attribute chain/list len of 0, 1, 2 and 3 at minimum, with negative and positive cases.